bastion-executor/src/load_balancer.rs

```diff
@@ -47,46 +51,112 @@ impl LoadBalancer {
     }
 }
 
+/// Maximum number of core supported by mordern computers.
```

Comment:
```suggestion
/// Maximum number of core supported by modern computers.
```

it is ok, i missed this.